Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var string[]
+     */
+    protected static $formats = array('json', 'xml', 'yml');
```

Comment:
@greg0ire do you mean that it will be configurable from a config file in the future?
So I need to add something like this:
```php
        $node
            ->children()
                ->arrayNode('serializer')
                    ->addDefaultsIfNotSet()
                    ->children()
                        ->arrayNode('formats')
                            ->prototype('scalar')->end()
                            ->defaultValue(array('json', 'xml', 'yml'))
                        ->end()
                    ->end()
                ->end()
            ->end()
        ;
```

Or just set it in the `SonataCoreExtension` like this:
```php
BaseSerializerHandler::setFormats($config['serializer']['formats']);
```
And since we are adding a deprecating warning why do we avoid from its message. Because if we are going to change it in the future shouldn't we show this warning message to tell them to change it, or configure it from configuration file?